tutorials/tour/basics.md

```diff
@@ -0,0 +1,207 @@
+---
+layout: tutorial
+title: Basics
+
+disqus: true
+
+tutorial: scala-tour
+num: 2
+next-page: unified-types
+previous-page: tour-of-scala
+---
+
+In this page, we will cover basics of Scala.
+
+## Variables
+
+You can define variables with `var` keyword.
+
+```
+var x = 1 + 1
+x += 1
+```
+
+Often, you want your variables to be immutable. You can use `val` keyword in that case.
```

Comment:
I'm not sure that _immutable_ should be used at all. I (maybe incorrectly) expect that newcomers will be later confused by the whole immutable reference vs. immutable object argument, since it's not immediately intuitive that immutability at the variable definition level doesn't guarantee immutability of _what_ is referenced by that variable name.

@a1russell your definition seems good to me in spirit, but I'd rephrase the second part as

> `val`s cannot be re-assigned and they will always reference the same value. If you need to re-assign a variable, you can use the `var` keyword. It is considered good practice to reduce the use of `var`s to the bare minimum.
  ```
  var y = 1 + 2
  y += 4
  //y now is equal to 7
  ```